packages/manager-components/src/components/clipboard/translations/index.ts

```diff
@@ -0,0 +1,13 @@
+import i18next from 'i18next';
+
+import fr_FR from './Messages_fr_FR.json';
+
```

Comment:
Don't forget to modify this file once you have the translations to include all the languages